sample/db/samples/payment_methods.rb

```diff
@@ -2,7 +2,6 @@
   {
     :name => "Credit Card",
     :description => "Bogus payment gateway for development.",
-    :environment => "development",
```

Comment:
@jhawthorn awesome, thanks! I'll try to squash that early next week so we can get this merged in.